jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DefaultServletTest.java

```diff
@@ -114,6 +115,11 @@ public void init() throws Exception
         connector = new LocalConnector(server);
         connector.getConnectionFactory(HttpConfiguration.ConnectionFactory.class).getHttpConfiguration().setSendServerVersion(false);
 
+        ServerConnector serverConnector = new ServerConnector(server);
+        serverConnector.setPort(8080);
```

Comment:
Don't use a hardcoded port, esp 8080.

Reply:
oops yeah I did that for debugging, I will remove.